serverless/src/util/jsonToCsv.js

```diff
@@ -0,0 +1,27 @@
+import { get } from 'sendero'
+import { unparse } from 'papaparse'
+
+/**
+ * @name jsonToCsv
+ * @description converts an array of JSON objects to a CSV
```

Comment:
While `@description` is probably the right way to do this, it isn't how we have written the descriptions in the comment blocks of anything else. `@name` also isn't used anywhere else. Comments should be consistent across the repo

Reply:
updated via https://github.com/nasa/earthdata-search/pull/1630/commits/52aa0cd4438b447be66e37562730390d13a473c8